packages/app-project/src/screens/ClassifyPage/components/WorkflowAssignmentModal/WorkflowAssignmentModal.spec.js

```diff
@@ -21,13 +21,15 @@ describe('Component > WorkflowAssignmentModal', function () {
     closeFnSpy = sinon.spy()
     dismissSpy = sinon.spy()
     sinon.stub(nextRouter, 'useRouter').callsFake(() => router)
+    useTranslationStub = sinon.stub(i18n, 't').callsFake((key) => key)
```

Comment:
Not a blocking comment, but we should be cautious about using stubs in tests. They block refactors by tying your tests to the internal implementation of a feature. #2719 has examples of this, where the subject viewers stub `componentDidMount`, which then blocked refactoring them from classes to functional components.

Reply:
Good to know thanks!